docs/latest/google.resumable_media.requests.html

```diff
@@ -808,8 +808,17 @@
 
```

Comment:
I preferred this over a `gh-pages` branch:

![screenshot from 2017-05-01 09-35-07](https://cloud.githubusercontent.com/assets/520669/25585871/7f8ab78a-2e51-11e7-84e5-f68f549f7cf8.png)